tests/Handler/AddPrefixCommandHandlerTest.php

```diff
@@ -84,7 +84,11 @@ protected function tearDown()
     {
         chdir($this->workingDirectory);
         $filesystem = new Filesystem();
-        $filesystem->remove($this->tempDir);
+        if (!defined('PHP_WINDOWS_VERSION_BUILD')) {
+            $filesystem->remove($this->tempDir);
+        } else {
+            exec(sprintf("rd /s /q %s", escapeshellarg($this->tempDir)));
```

Comment:
I'm not sure I'd call it a work around, it's a one line command which does the same thing as spending tens of lines writing something with a directory iterator. The Filesystem component is up to 3.2 and doesn't seem to handle this case.